tests/test_store.py

```diff
@@ -113,9 +114,37 @@ def test_getFileAnnotations(self, synapse_store):
 
         assert expected_dict == actual_dict
 
-    def test_annotation_submission(self, synapse_store, helpers, config: Configuration):
-        manifest_path = "mock_manifests/annotations_test_manifest.csv"
-
+    @pytest.mark.parametrize('only_new_files',[True, False])
+    def test_file_entityIds(self, helpers, synapse_store, only_new_files):
```

Comment:
could you change the function name to `test_get_file_entityIds`

Reply:
sure